src/sentry/rules/actions/integrations/create_ticket/utils.py

```diff
@@ -130,17 +131,11 @@ def create_issue(event: GroupEvent, futures: Sequence[RuleFuture]) -> None:
             try:
                 response = installation.create_issue(data)
             except Exception as e:
-                logger.info(
-                    "%s.rule_trigger.create_ticket.failure",
-                    provider,
-                    extra={
-                        "rule_id": rule_id,
-                        "provider": provider,
-                        "integration_id": integration.id,
-                        "error_message": str(e),
-                        "exception_type": type(e).__name__,
-                    },
-                )
+                if isinstance(e, IntegrationFormError):
+                    # Most of the time, these aren't explicit failures, they're
+                    # some misconfiguration of an issue field - typically Jira.
+                    lifecycle.record_halt(e)
```

Comment:
do we still want to record the extra logging info?

Reply:
It should already be logged via the lifecycle calls on lines 127-129, it just wasn't captured in the diffs.